packages/design-system/src/components/OcTag/OcTag.vue

```diff
@@ -30,6 +30,10 @@ export interface Props {
    * @default false
    */
   rounded?: boolean
+  /**
+   * @docs The color of the tag.
+   */
+  color?: 'default' | 'warning' | 'custom-purple'
```

Comment:
I'd stick with the color role schema as we know it from material: `primary`, `secondary` (default) and `tertiary`.